src/checkingBaseball.js

```diff
@@ -0,0 +1,50 @@
+import { Console } from '@woowacourse/mission-utils';
+
+export default function checkingBaseball(player, computer) {
+	try {
+		const BALL = checkBall(player, computer);
+		const STRIKE = checkStrike(player, computer);
+		printResult(BALL, STRIKE);
+		if (STRIKE === 3) {
+			return false;
+		}
+		return true;
+	} catch (error) {
+		Console.print(error);
+	}
+}
+
+const checkBall = function countingBall(player, computer) {
+	let ballCount = 0;
+	for (let i = 0; i < 3; i++) {
+		if (computer.includes(player[i])) {
+			ballCount += 1;
+		}
+	}
+	return ballCount;
+};
+
+const checkStrike = function countingStrike(player, computer) {
+	let strikeCount = 0;
+	for (let i = 0; i < 3; i++) {
+		if (player[i] === computer[i]) {
+			strikeCount += 1;
+		}
+	}
```

Comment:
```suggestion
	computer.forEach((num, index) =>  {
		if (num === player[index]) {
			strikeCount += 1;
		}
	})
```

저는 for문은 도중에 break나 return을 걸어야할 때만 사용하고 그 이외에는 고차함수를 사용하는 것 같아요!!
저는 처음에 for / while / forEach / map / filter / reduce가 정말 헷갈렸는데 하나하나 정리해보니 도움이 많이 되었던 거 같아요!!